nodes/core/io/21-httpin.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013,2014 IBM Corp.
+ * Copyright 2013,2014,2015 IBM Corp.
```

Comment:
Format for multiple dates is:

   Copyright START_YEAR, END_YEAR IBM Corp

not a list of every year